framework/include/invokee.hpp

```diff
@@ -30,24 +30,27 @@ namespace gvk
 		/**
 		 * @brief Constructor which automatically generates a name for this object
 		 */
-		invokee()
+		invokee(int pExecutionOrder = 0)
```

Comment:
```suggestion
		invokee(int aExecutionOrder = 0)
```

Reply:
My mistake. (work uses p and it got mixed up)